Framework/Crystal/src/PredictPeaks.cpp

```diff
@@ -101,6 +101,12 @@ void PredictPeaks::init() {
                   "values in the HKLPeaksWorkspace to the nearest integers if "
                   "checked.\n"
                   "Keep unchecked to use the original values");
+
+  declareProperty("PredictPeaksOutsideDetectors", false,
+                  "Use an extended detector space (if defined for the"
+                  " instrument) to predict peaks which do not fall onto any"
+                  "detector. This may produce a very high number of results.");
+
```

Comment:
Will inserting the new property here change the algorithm's Python signature so that old scripts might stop working? I mean, somebody might have
```python
PredictPeaks(inputWS, 0.1, 100, 1, 100, Primitive, False, hklWS, True, False, 'outputWS')
```
in their scripts.